package.json

```diff
@@ -35,5 +35,8 @@
     "stylelint-config-standard": "~18.0.0",
     "surge": "~0.20.3",
     "redux-mock-store": "~1.2.2"
+  },
+  "dependencies": {
+    "@redhat-cloud-services/frontend-components": "^6.0.7"
```

Comment:
We currently use v3, when i tried to use v6 I got webpack compile errors as it didnt find some pf
I tried using v3 with scalprum and it worked fine 
`npm i` shouldn't be an issue with this as long as you use the supported v6, or run `npm i --legacy-peer-deps `